docs/en/docs/kant/best-practice/modbus-device-demo.md

```diff
@@ -0,0 +1,135 @@
+---
+MTPE: WANG0608GitHub
+Date: 2024-07-30
+---
+
+# Instructions to Connect a Modbus Device
+
+This article introduces how to connect an end devices based on the Modbus protocol to DCE 5.0 Cloud Edge Collaboration and interact with the cloud.
+A simulated device is used as an example to introduce the overall implementation process.
+
+Connect Process:
+
+```mermaid
+graph TD
+    A(Start) --> B(Connecting end devices to edge nodes)
+    B --> C(Creating device models)
+    C --> D(Creating end devices)
+    D --> E(Deploying the device Mapper)
+    E --> F(End)
+classDef plain fill:#ddd,stroke:#fff,stroke-width:1px,color:#000;
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef cluster fill:#fff,stroke:#bbb,stroke-width:1px,color:#326ce5;
+
+class A,F plain
+class B,C,D,E cluster
+
+click B "https://docs.daocloud.io/en/kant/best-practice/modbus-device-demo.html#connecting-end-devices-to-edge-nodes"
+click C "https://docs.daocloud.io/en/kant/best-practice/modbus-device-demo.html#creating-device-models"
+click D "https://docs.daocloud.io/en/kant/best-practice/modbus-device-demo.html#creating-terminal-devices"
+click E "https://docs.daocloud.io/en/kant/best-practice/modbus-device-demo.html#deploying-the-device-mapper"
+```
+
+## Preparation work
+
+- Simulated end device with the Modbus protocol: In this article, the end device is simulated as a deployment.
+
+    Simulated device image: **release-ci.daocloud.io/kant/modbusmock:v0.1.0**
+
+- Device Mapper: Developed based on the KubeEdge DMI framework, it can collect data for Modbus protocol devices. This device is only suitable for the simulated device mentioned above.
```

Comment:
```suggestion
- Mapper Device: Developed based on the KubeEdge DMI framework, it can collect data for Modbus protocol devices. This device is only suitable for the simulated device mentioned above.
```